css-shared-element-transitions/index.bs

```diff
@@ -298,241 +299,302 @@ Single-page API:
+
+        Issue: How should input be handled when in this state? The last frame presented
+        to the user will not reflect the DOM state as it asynchronously switches to
+        the new version.
+
+    1. [=Queue a global task=] on the [=DOM manipulation task source=],
+        given |transition|'s [=relevant global object=],
+        to execute the following steps:
+
+            Note: A task is queued here because the texture read back in [=capturing the image=] may be async,
+                although the render steps in the HTML spec act as if it's synchronous.
+
+        1. Let |timedOut| be false.
+
+        1. Let |userP| be the result of [=/invoking=] |transition|'s [=SameDocumentTransition/prepare callback=].
+
+        1. [=promise/React=] to |userP|:
+
+            * If |userP| does not settle within an implementation-defined timeout, then:
+
+                1. Set |timedOut| to true.
+
+                1. If |preparationFailed| is true,
+                    then [=abandon the page transition=] |transition| with an "{{AbortError}}" {{DOMException}},
+                    and abort these steps.
+
+                1. Otherwise, [=abandon the page transition=] |transition| with a "{{TimeoutError}}" {{DOMException}}.
+
+            * If |userP| was fulfilled, then:
+
+                1. If |timedOut| is true, then return.
+
+                1. Stop suppressing rendering opportunities for |transition|'s Document.
+
+                    Note: Resuming rendering opportunities is delayed until fulfillment of
+                    |userP| to allow asynchronous loading of the incoming DOM.
+
+                1. If |preparationFailed| is true,
+                    then [=abandon the page transition=] |transition| with an "{{AbortError}}" {{DOMException}},
+                    and return.
+
+                1. Set |usedTransitionTags| to a new [=/set=].
+
+                1. [=list/For each=] |element| of every {{Element}} and [=pseudo-element=] connected to |document|,
+                    in [paint order](https://drafts.csswg.org/css2/#painting-order):
+
+                    Issue: The link for "paint order" doesn't seem right.
+                        Is there a more canonical definition?
+
+                    1. Let |transitionTag| be the [=computed value=] of 'page-transition-tag' for |element|.
+
+                    1. If |transitionTag| is ''page-transition-tag/none'',
+                        or |element| is [=element-not-rendered|not rendered=],
+                        then [=continue=].
+
+                    1. If any of the following is true:
+
+                        * |usedTransitionTags| [=list/contains=] |transitionTag|.
+
+                        * |element| is not |element|'s [=tree/root=]
+                            and |element| does not have [=layout containment=].
+
+                        * |element| is not |element|'s [=tree/root=]
+                            and |element| does prevent [=fragmentation=].
```

Comment:
```suggestion
                            and |element| allows [=fragmentation=].
```

Ditto here. We need to ensure the content is not fragmented to make it easier to snapshot. We need an issue to clearly define what the developer needs to do in this case. Like adding "break-inside:avoid" to the element's style.